massgov/mayflower
# [Documentation] a11y Markup Checklist

## Description

Set up the markup checklist for accessibility as developers' reference.

Cover topics developers tend to misuse elements and attributes, which have been seen in Mayflower.

Temporally, access to the doc is set up at patternlab/README.md. Still searching for better entrance to the doc.

## Related Issue / Ticket

- [JIRA issue](https://jira.mass.gov/browse/DP-13704)

## Steps to Test


The documentation can be accessed from: 

- https://github.com/massgov/mayflower/tree/docs-DP-13704_a11y-markup-checklist/patternlab
- https://github.com/massgov/mayflower/blob/docs-DP-13704_a11y-markup-checklist/patternlab/docs/a11y-checklist/a11y-checklist.md

1. Check for: 

- flow of the content,
- practical use of the document,
- any grammatical errors

## Screenshots
Use something like [licecap](http://www.cockos.com/licecap/) to capture gifs to demonstrate behaviors.


## Additional Notes:

Anything else to add?

#### Impacted Areas in Application


* 

#### @TODO


*

#### Today I learned...
